### PR TITLE
Country table filled by the code

### PR DESCRIPTION
**Pull request by 2208Abhinav**
This is the code to fill country table in lists with country names and their codes.

### CODE REVIEW

File: sql/country_table.sql [hunk 1]
```diff
@@ -0,0 +1,257 @@
+<?php
+
+include_once("../globals.php");
```
Comment (teryhill) [inline sql/country_table.sql]: Check this and see if it works from this directory.

Comment (2208Abhinav) [reply sql/country_table.sql]: Checking....

Comment (2208Abhinav) [reply sql/country_table.sql]: I had to change it to ../interface/globals.php

Comment (2208Abhinav) [reply sql/country_table.sql]: and extension to .php because I am running the script manually

Comment (teryhill) [reply sql/country_table.sql]: Yep

**Comment by teryhill** [comment]
Give it a .php extension. Since it is actually a program and not just a sql script

**Comment by 2208Abhinav** [comment]
Changed the extension

**Comment by 2208Abhinav** [comment]
I  think this script should be put in such a way that it runs when the software is being installed

**Comment by teryhill** [comment]
Let the user decide. if they need them all then they can add them. Don't force the user to do something. Like going to gitlab

**Comment by 2208Abhinav** [comment]
Okay I understand. I think you don't feel happy about the gitlab decision. :thinking:

**Pull request merged**